pkgs/applications/science/math/cplex/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, coreutils, requireFile, patchelf, makeWrapper, openjdk, gtk2, xorg, glibcLocales, releasePath ? null }:
+
+# To use this package, you need to download your own cplex installer from IBM
+# and override the releasePath attribute to point to the location of the file.  
+#
+# Note: cplex creates an individual build for each license which screws
+# somewhat with the use of functions like requireFile as the hash will be
+# different for every user.
+
+stdenv.mkDerivation rec {
+  name = "cplex-${version}";
+  version = "128";
+  
+  src =
+    if builtins.isNull releasePath then
+      throw ''
+        This nix expression requires that the cplex installer is already
+        downloaded to your machine. Get it from IBM and override the
+        releasePath attribute to point to the location of the file. 
+      ''
+    else
+      releasePath;
+
+  nativeBuildInputs = [ coreutils makeWrapper ];
+  buildInputs = [ openjdk gtk2 xorg.libXtst glibcLocales ];
+
+  phases = "patchPhase buildPhase installPhase fixupPhase";
```

Comment:
I meant to remove this line completely, these phases are set like this automatically

Reply:
How can I disable unpackPhase? I tried to define an empty one, but it did not work (and it complains it does not know how to unpack the cplex bin archive if I do not remove the phase completely).

Reply:
Maybe `unpackPhase = "cp $src ."` will work

Reply:
cp $src $name